### PR TITLE
Refer to samples folder in README

### LINKED ISSUE #249: Add small language examples to the README
Just to show the gist of the language for the reader.

### PR DESCRIPTION
**Pull request by rexim**
Close #249

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ language that recognizes Church encoded structures.
...
 
+### Samples ###
+
+For language samples and examples please visit [docs/samples/][samples-dir] directory. To execute a sample run:
+
+    $ sbt "run <path-to-sample>"
+
+from the root of the source code directory. For example, to run the 15 first fibonacci numbers sample run:
+
+    $ sbt "run docs/samples/fib.mgn"
```
Comment (ForNeVeR) [inline README.md]: I suggest to put such blocks into `console` environment such as:

``````
```console
$ sbt "run <path-to-sample>"
```
``````

This way GitHub will color them more nicely.

Comment (rexim) [reply README.md]: Wow! Didn't know that such thing exists! Let me try it...

Comment (rexim) [reply README.md]: @ForNeVeR looks cool! Thanks!

**Review by ForNeVeR — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -11,11 +11,29 @@ language that recognizes Church encoded structures.
...
 
+### Samples ###
+
+For language samples and examples please visit [docs/samples/][samples-dir] directory. To execute a sample run:
+
+```console
+$ sbt "run <path-to-sample>"
+```
+
+from the root of the source code directory. For example, to run the 15 first fibonacci numbers sample run:
```
Comment (ForNeVeR) [inline README.md]: I suggest to replace the first "run" in this line with "print".

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

**Review by keddelzz — APPROVED** [review]
:+1: LGTM

File: README.md (hunk 2 above)
Comment (rexim) [reply README.md]: "To print 15 first fibonacci sample"? What "printing a sample" is supposed to mean in this context? I'd replace it with `execute`. Let me do that.

**Review by ForNeVeR — APPROVED** [review]

**Comment by ForNeVeR** [comment]
AppVeyor failed to start the job again for some reason, but that shouldn't block the merge.

Also, probably something is strange with our AppVeyor setup; such periodic failures are unusual. I'll take a look at our setup later.

**Comment by rexim** [comment]
@ForNeVeR hope you don't mind that I created an issue for that and assigned to you.

**Comment by ForNeVeR** [comment]
@rexim thanks!

**Comment by rexim** [comment]
@ForNeVeR @keddelzz thanks for the review guys!

**Pull request merged**